### PR TITLE
Correct v5.1 doc errors

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Correct v5.1 doc errors

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -351,9 +351,8 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 ### `flow-round-by-digit` <span class="version-mark">从 v5.1 版本开始引入</span>
 
-+ 打开 `flow-round-by-digit`
 + 默认值：3
-+ PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
++ PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `1001000` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
```
Comment (TomShawn) [inline pd-configuration-file.md:355]: @nolouch PTAL, thanks!

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ tiup cluster upgrade <cluster-name> v5.1.0
 
 - Normal：同步任务正常进行，checkpoint-ts 正常推进。
 - Stopped：同步任务停止，由于用户手动暂停 (pause) changefeed。处于这个状态的 changefeed 会阻挡 GC 推进。
-- Error：同步任务报错，由于某些可恢复的内部错误导致同步无法继续进行，处于这个状态的 changefeed 会不断尝试继续推进，直到状态转为 Normal。处于这个状态的 changefeed 会阻挡 GC 推进。
+- Error：同步任务报错，由于某些可恢复的错误导致同步无法继续进行，处于这个状态的 changefeed 会不断尝试继续推进，直到状态转为 Normal。处于这个状态的 changefeed 会阻挡 GC 推进。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:82]: This has been confirmed with @amyangfei .

File: pd-control.md [hunk 3]
```diff
@@ -450,7 +450,7 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 
 - `store-limit-mode` 用于控制 store 限速机制的模式。主要有两种模式：`auto` 和 `manual`。`auto` 模式下会根据 load 自动进行平衡调整（实验性功能）。
 
-- PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
+- PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `101000`。默认值为 `3`。该配置替换了 `trace-region-flow`。
```
Comment (TomShawn) [inline pd-control.md:453]: This has been confirmed with @nolouch .

File: information-schema/information-schema-deadlocks.md [hunk 4]
```diff
@@ -109,9 +109,9 @@ select * from information_schema.deadlocks;
 >
 > 目前，`DEADLOCKS` 表暂不支持收集可重试的死锁错误相关的信息。
 
-当事务 A 被另一个事务已经持有的锁阻塞，而事务 B 直接或间接地被当前事务持有的锁阻塞，将会引发一个死锁错误。这里：
+当事务 A 被另一个事务 B 已经持有的锁阻塞，而事务 B 直接或间接地被当前事务 A 持有的锁阻塞，将会引发一个死锁错误。这里：
```
Comment (TomShawn) [inline information-schema/information-schema-deadlocks.md:112]: This has been confirmed with @MyonKeminta .

**Review by CharLotteiu — APPROVED** [review]

**Comment by Liuxiaozhen12** [comment]
This PR does not need to be translated because all the errors in docs-cn in this PR were not found in docs.
/remove-translation doing
/translation no-need

**Pull request merged**